CASMCMS-9423/CASMCMS-9448: Remove/update docs for removal of CPS/DVS, long-deprecated csm-config playbooks (#5994)
Cray-HPE/docs-csm: operations/boot_orchestration/Session_Templates.md

`rootfs_provider` identifies the mechanism that provides the root file system for the node.

> In the case of the [Cray Operating System (COS)](../../glossary.md#cray-operating-system-cos) image, the `rootfs_provider` is HPE's

Author: ```suggestion
In the case of the [User Services Software (USS)](../../glossary.md#user-services-software-uss) image, the `rootfs_provider` is HPE's
```

In the case of the [Cray Operating System (COS)](../../glossary.md#cray-operating-system-cos) image, the `rootfs_provider` is HPE's
[iSCSI SBPS (Scalable Boot Content Projection Service)](../iscsi_sbps/iscsi_sbps.md) to deliver the content.

> CPS projects the root file system onto the nodes as a SquashFS image. This is provided via an overlay file system which is set up in dracut.

Reviewer: Maybe leave this sentence in, but just change `CPS` to `SBPS`.

Author: ```suggestion
SBPS projects the root file system onto the nodes as a SquashFS image. This is provided via an overlay file system which is set up in dracut.
```

> 

Author: ```suggestion
SBPS projects the root file system onto the nodes as a SquashFS image. This is provided via an overlay file system which is set up in dracut.

```

In the case of the [Cray Operating System (COS)](../../glossary.md#cray-operating-system-cos) image, the `rootfs_provider` is HPE's

> [iSCSI SBPS (Scalable Boot Content Projection Service)](../iscsi_sbps/iscsi_sbps.md) to deliver the content.

Author: ```suggestion
[iSCSI SBPS (Scalable Boot Content Projection Service)](../iscsi_sbps/iscsi_sbps.md).
```
